src/fragments/lib/analytics/flutter/getting-started/10_preReq.mdx

```diff
@@ -1,7 +1,5 @@
 * [Install and configure Amplify CLI](https://docs.amplify.aws/cli/start/install)
 * A Flutter application targeting Flutter SDK >= 2.0.0 (stable version) with Amplify libraries integrated
-    * An iOS configuration targeting at least iOS 11.0
```

Comment:
I feel like it doesn't make sense to remove those for now. Especially since we have a difference between using these minimum target platforms between categories. Also, because developers might choose to directly go to a category and use it, rather than go through the setup. Same applies for Android here.

Reply:
I can always revert them if you think these are valuable. 👍🏼